docs/learn-faq.md

```diff
@@ -81,6 +81,88 @@ This script helps you quickly identify which validator is the lowest-staked and
+1) blockchains need some way to adapt and evolve and 2) the only measure of value
+that is intrinsic to a blockchain is its native coin and therefore the best
+measure of stakeholders for the system are the token holders. There are of
+course caveats to the above two observations and that is why Polkadot's governance
+is still in its experimental phases. Polkadot takes the approach that it will try
+out best methods and observe how they work and tweak them until they're optimal.
+
+### Polkadot's governance is plutocratic
+
+Since Polkadot uses "coin-voting" that allows token holders to vote based on the
+amount of their holdings, some claim that Polkadot must be plutocratic. 
+
+However, Polkadot's governance is actually more fine-grained than this and has
+built in mechanisms that increase resiliency over plutocracy and cartels. One of
+these mechanisms is the council, an elected body that can make motions that will
+be voted on as refernda with a bias toward passing. The council acts as the voice
+for passive token holders or other users of the system that cannot vote. Another
+mechanism that can be employed by any token holder is called _conviction voting_
+and allows any toke holder to amplify their vote for stronger voting power. These
+mechanisms, and more, provide counter weights to the tendency for a simple token
+voting system to devolve into a plutocracy.
+
+For more on Polkadot's governance please see the [governance page](learn-governance).
+
+### Eth2.0 is better because it has 200,000+ validators and Polkadot has 1000
+
+The answer to this myth is still at this point totally unclear. The argument is
+that Eth2.0 will be more decentralized because it supports more validators than
+Polkadot. On Eth2.0, the barriers to entry for being a validator can be lower
+because the requirement is only 32 ETH, while on Polkadot it can be many multiples
+of this. On the flip side Polkadot has a more robust nomination system for electing
+the validator set that Eth2.0 lacks. It is still uncertain whether a decentralized
+network can support the network bandwidth and overhead of consensus messages being
+gossiped among hundreds of thousands of nodes. Polkadot's estimation of 1000
+validators is a more realistic estimation and if its possible to scale to more
+then it's possible for on-chain governance to enable it in the future.
+
+### The auction mechanism for parachains implies they are ephemeral
+
+The myth is that the parachain auction mechanism for Polkadot implies that
+all applications will be ephemeral, or, worse, eventually be bought out by more
+wealthy bidders. 
+
+The resolution of this myth is to understand Polkadot's tiered applications offering
+that includes parachains, parathreads, and smart contracts. Parachains can
+move to being a parathread and only producing a block when necessary. Parachain
+slots are only needed by the applications with heaviest load that need block-by-block
```

Comment:
I don't think this is a good enough explanation for defi projects, which are all everyone is talking about in Eth right now. Until the next fad comes along, Eth == defi, so we must address this from that angle. 

Defi MUST have block by block production because you'll be trading across parachains, potentially. So if someone builds a synthetix, and someone builds a uniswap, and someone builds a compound, unless those are all on the same chain (governance problems) they need to communicate in sync. As we talked about in teched the other day, there's XCMP overhead to think about, custom token functionality to consider (can't move iBTC to the uniswap chain with Balances SPREE because iBTC has freezing logic in it), and other details that make Polkadot wholly unsuitable for these types of deployments. 

Just imagine MakerDAO - the whole eth financial ecosystem is resting on the shoulders of that giant. If that loses sync, people lose deposits and collateral, and all of defi collapses, it's a black swan waiting to happen. Maker, despite being a huge org, would never deploy on a parachain because it can be bought out or lost. Just the possibility being there, however unlikely, would turn them away. But they would deploy on a system-level contract chain, if one existed.

Rather than "it'll be fine"-ing it, I'd sooner go into detail about app specific blockchains and enterprises running parachains, and state that everyone else will probably end up on either a smart contract parachain made permanent by the governance process, bypassing the lease, or in bridges to Ethereum and Tezos.